app/components/gleam-logo.ts

```diff
@@ -0,0 +1,114 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { Fit, Layout, Rive } from '@rive-app/canvas';
+import * as Sentry from '@sentry/ember';
+
+interface GleamLogoSignature {
+  Element: HTMLDivElement;
+
+  Args: {
+    height: number;
```

Comment:
Let's try to avoid needing a number to be passed in and see if we can just have it fit the container / parent element